components/fxa-client/src/http_client.rs

```diff
@@ -543,11 +543,7 @@ pub fn get_keys_bundle(config: &Config, hkdf_sha256_key: &[u8]) -> Result<Vec<u8
     let req = HawkRequestBuilder::new(Method::Get, keys_url, hkdf_sha256_key).build()?;
     let client = Client::new();
     let resp: serde_json::Value = client.make_request(req)?.json()?;
-    let bundle = hex::decode(
-        &resp["bundle"]
-            .as_str()
-            .ok_or_else(|| ErrorKind::KeyGenerationFailed)?,
-    )?;
+    let bundle = hex::decode(&resp["bundle"].as_str().unwrap())?;
```

Comment:
Why the switch to `unwrap` here? It seems like continuing to return some sort of error (even if it's a less specific one) would be better than panicking.